### PR TITLE
Fix/fix user api response

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[會員制] 會員專區的會員身份 API 串接](https://app.asana.com/0/1203699264568808/1204362012792382/f)
讓 response 符合文件

# Notice

# Dependency

### CODE REVIEW

**Review by Aylie-Chou — APPROVED** [review]
LGTM

File: models/model_user.go [hunk 1]
```diff
@@ -48,6 +48,7 @@ type Role struct {
 	ID        string    `json:"id"`
 	Name      string    `json:"name"`
 	NameEn    string    `json:"name_en"`
+	Key       string    `json:"key"`
```
Comment (howar31) [inline models/model_user.go:51]: please update [Notion schema doc](https://www.notion.so/twreporter/roles-b7cda5d35974496dbdb07cfea0b52387?pvs=4) accordingly

**Review by howar31 — CHANGES_REQUESTED** [review]
I do agree the add of key column.  Please also update the related schema docs.  Thanks

File: models/model_user.go (hunk 1 above)
Comment (liruchen32) [reply models/model_user.go:51]: fixed

**Review by howar31 — APPROVED** [review]
LGTM

**Pull request merged**